ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude

```diff
@@ -305,6 +317,26 @@ public void ValidateAndSetIgnoreUnexpectedElementsAndAttributesFromString(string
     this.IgnoreUnexpectedElementsAndAttributes = boolValue;
 }
 
+/// <summary>
+/// Set the MakeTypesInternal property with the given value.
+/// </summary>
+/// <param name="stringValue">The value to set.</param>
+public void ValidateAndSetMakeTypesInternalFromString(string stringValue)
```

Comment:
```suggestion
public void ValidateAndMakeTypesInternalFromString(string stringValue)
```

Reply:
Here I was following the naming pattern of the other validation methods, which is `ValidateAndSet<PropertyName>FromString`, and in this case the property name happens to be `MakeTypesInternal`. Another example in the code is `ValidateAndSetUseDataServiceCollectionFromString`